socorro/external/es/supersearch.py

```diff
@@ -169,6 +171,15 @@ def get(self, **kwargs):
         # Find the indices to use to optimize the elasticsearch query.
         indices = self.get_indices(params["date"])
 
+        if "%" in self.context.get_index_template():
```

Comment:
If there's no `%` in the template, then it's not date based and the cutoff isn't pertinent.

At the moment, this is only an issue with one of the tests. I figured it's easier to add this check here than try to figure out whether we should support non-date-based templates because that has big ramifications on test setup and a bunch of other stuff.